src/controllers/getURL.js

```diff
@@ -0,0 +1,18 @@
+import Url from '../models/UrlShorten';
+
+const getUrlAndEditCount = async (req, res) => {
+  try {
+    const url = await Url.findOne({ urlCode: req.params.urlCode });
+    if (url) {
+      url.click_count += 1;
+      await url.save();
+      return res.redirect(url.long_url);
+    } else {
```

Comment:
There is no need wrapping this statement in an else block since we already returned a response inside the if block.

Reply:
Attend to this then

Reply:
I will be attending to this shortly